adapt-and-extend/theming/resourceregistry.rst

```diff
@@ -124,166 +236,258 @@ In ``mockup/patterns/structure/js/views/actionmenu.js``:
 
-The following are for pre-compiled bundles and are automatically set when the bundle is built Through-The-Web:
+jscompilation
+    URL of the compiled and minified JavaScript file.
 
-- jscompilation: URL of the compiled and minified JavaScript file.
+csscompilation
+    URL of the compiled and minified CSS file.
 
-- csscompilation: URL of the compiled and minified CSS file.
+last_compilation
+    Date of the last compilation time.
+    The value of this option is automatically used as version parameter for cache-busting in production mode.
+    (eg. ``plone-logged-in-compiled.min.js?version=2015-05-07%2000:00:00.000003``)
 
-- last_compilation: Date of the last compilation time. Used as version parameter for caching
-  (eg. plone-logged-in-compiled.min.js?version=2015-05-07%2000:00:00.000003)
 
+.. _bundles_compiling_bundles:
 
-Bundle compilation
-------------------
+Compiling Bundles
+-----------------
+
+There are three ways to provide a compiled version of a bundle for production:
+
+**Compile the bundle Through-The-Web and store it in the ZODB**
+
+When using this option, all an add-on developer or an integrator needs to do is register a bundle with the "compile" option set to ``True``.
+In the Plone 5 Resource Registry control panel, a button will be available to compile the bundle.
+Pressing this button will compile the bundle and store it for production delivery.
 
-In order to provide a compiled version for the production mode there are three possibilities:
+**Compile the bundle from the command line**:
 
-- Compile Through-The-Web and store on the ZODB.
-  This is done via the resource control panel.
+Plone provides a script which will compile a specific bundle available in the resource registry.
+To use this option, you must specifically request the script in your buildout.
+Add a new part called "resources" and list it in your buildout "parts", then re-run buildout.
+You will find the ``plone-compile-resources`` script in your buildout ``bin`` directory.
 
-- Compile with a generated Grunt file: ``./bin/plone-compile-resources --site-id=myplonesite --bundle=mybundle``
+.. code-block:: ini
 
-- Create your own compilation chain: Using the tool you prefer create a compiled version of your bundle with the correct URLs.
+    [resources]
+    recipe = zc.recipe.egg
+    eggs = Products.CMFPlone
+    scripts = plone-compile-resources
 
+Once the script has been created you may invoke it.
+You will need to provide options indicating the ID of the Plone site in which your package is installed, and the name of the bundle you wish to compile:
+
+    ./bin/plone-compile-resources --site-id=myplonesite --bundle=mybundle
+
+This script will start up your Plone site, extract the required information and compile the bundle.
+Because of this, if you are not using ZEO, you will need to stop your instance before running this script.
```

Comment:
`` if you are not using ZEO`` is confusing, you need to stop one instance anyway.